api.md

```diff
@@ -887,12 +888,51 @@ SELECT detach_tablespaces('conditions');
 ```
 
 ---
+## distributed_exec() [](distributed_exec)
+
+This command is used on an access node to execute a SQL command across
+the data nodes of a distributed database. This command can be used to,
+e.g., create cluster-wide objects across all instances of a
```

Comment:
Think we should reserve "cluster" for "PostgreSQL cluster" and use "distributed database". In this case, you can just remove "cluster-wide" and just write "...create objects on all connected data nodes..." and avoid the extra note below.

Reply:
FWIW, `cluster-wide` here refers to the PostgreSQL terminology used in the official docs, like here: https://www.postgresql.org/docs/12/catalog-pg-authid.html. But yeah, the term is very overloaded.